bundle/tests/loader.go

```diff
@@ -29,13 +32,15 @@ func loadTarget(t *testing.T, path, env string) *bundle.Bundle {
 }
 
 func loadTargetWithDiags(path, env string) (*bundle.Bundle, diag.Diagnostics) {
-	ctx := context.Background()
+	ctx := logdiag.InitContext(context.Background())
+	logdiag.SetCollect(ctx, true)
 	b, err := bundle.Load(ctx, path)
 	if err != nil {
 		return nil, diag.FromErr(err)
 	}
 
-	diags := phases.LoadNamedTarget(ctx, b, env)
+	phases.LoadNamedTarget(ctx, b, env)
+	diags := logdiag.FlushCollected(ctx)
```

Comment:
Why do we need to still return diags here while in other places we don't anymore?